xbmc/rendering/dx/DirectXHelper.h

```diff
@@ -124,6 +124,23 @@ namespace DX
     return name;
   }
 
+  constexpr std::string_view DXGIFormatToStringBasic(const DXGI_FORMAT format)
```

Comment:
To me, "Basic" doesn't really convey that it's an abbreviation, maybe DXGIFormatToShortString? or DXGIFormatToStringAbbr?

Reply:
Changed to `DXGIFormatToShortString`.

I don't see any downside to using constexpr. Value returned never changes (is created when return one of the literals "RGBA10", etc. that is const anyway). Probably compiler optimizes creating all constexp and at runtime switch only selects which is returned.

Reply:
I'm not a C++ guru and couldn't say. I understand what you're trying to do, but there are lots of rules for constexpr functions, that I don't fully understand, and the vs compiler doesn't seem to comply 100%, so it's useless as a validation.
Maybe ask in Slack #core?
I won't object either way, it builds and runs, so "good enough" I guess.